.gitignore

```diff
@@ -0,0 +1,6 @@
+.venv
+*.csv
+#data
+*.xlsx
+__pycache__
+.env
```

Comment:
and a newline after this one.

Reply:
Alright, Is there a rationale to this or it's considered as a convention

Reply:
Noted